s_tui/Sources/FreqSource.py

```diff
@@ -60,11 +60,11 @@ def __init__(self, is_admin):
         self.is_admin = is_admin
         self.is_available = True
 
-        self.top_freq = 0
+        self.top_freq = -1
         self.turbo_freq = False
         self.last_freq = 0
-        self.last_freq_list = [0] * len(psutil.cpu_freq(True))  # TODO compatiabiolty issues?
-        self.top_freq_list = [0] * len(psutil.cpu_freq(True))  # TODO compatiabiolty issues?
+        self.last_freq_list = [0] * len(psutil.cpu_freq(True))
```

Comment:
This can fail if psutil.cpu_freq() is not implemented

Reply:
fixed